woocommerce/wc-admin
# Add a new note about the variable product when the owner adds the fir…

Fixes #5934 

This PR adds a note with a link to the variable product documentation when the store owner adds his/her first product.

### Screenshots

![Screen Shot 2021-01-06 at 2 37 17 PM](https://user-images.githubusercontent.com/4723145/103827051-f1c5fe80-502c-11eb-9fdd-46dee0a271c5.jpg)


### Detailed test instructions:

1. Add a new product if you don't have any yet.
2. Install & activate "WP Crontrol" plugin
3. Navigate to Tools -> Cron Events and run `wc_admin_daily` job
4. Navigate to WooCommerce -> Home and confirm a new note with the title "Learn more about variable products"
